open-reac/src/main/java/com/powsybl/openreac/parameters/output/ReactiveSlackOutput.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac.parameters.output;
+
+import com.powsybl.ampl.converter.AmplConstants;
+import com.powsybl.ampl.converter.AmplException;
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.commons.util.StringToIntMapper;
+import com.powsybl.openreac.exceptions.IncompatibleModelException;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Nicolas Pierre <nicolas.pierre at artelys.com>
+ * Reactive slacks in load convention.
+ */
+public class ReactiveSlackOutput extends AbstractNoThrowOutput {
+
+    public static final int EXPECTED_COLS = 6;
+    private static final String SEP = ";";
+
+    public static class ReactiveSlack {
+        public final String busId;
+        public final String voltageLevelId;
+        public final double slack;
+
+        public ReactiveSlack(String busId, String voltageLevelId, double slack) {
+            this.busId = Objects.requireNonNull(busId);
+            this.voltageLevelId = Objects.requireNonNull(voltageLevelId);
+            this.slack = slack;
+        }
+
+        public String getBusId() {
+            return busId;
+        }
+
+        public String getVoltageLevelId() {
+            return voltageLevelId;
+        }
+
+        public double getSlack() {
+            return slack;
+        }
+    }
+
+    private final List<ReactiveSlack> slacks = new ArrayList<>();
+
+    public List<ReactiveSlack> getSlacks() {
+        return slacks;
+    }
+
+    @Override
+    public String getFileName() {
+        return "reactiveopf_results_reactive_slacks.csv";
+    }
+
+    @Override
+    public void read(Path path, StringToIntMapper<AmplSubset> amplMapper) {
+        List<String> investmentsLines;
```

Comment:
investment or reactive slack?

Reply:
Reactive slack is good wording. I'm removing the last bits.